Sources/XCTestDynamicOverlay/Internal/DefaultInitializable.swift

```diff
@@ -0,0 +1,44 @@
+protocol _DefaultInitializable {
+  init()
+}
+
+extension Array: _DefaultInitializable {}
+extension Bool: _DefaultInitializable {}
+extension Character: _DefaultInitializable { init() { self.init(" ") } }
+extension Dictionary: _DefaultInitializable {}
+extension Double: _DefaultInitializable {}
+extension Float: _DefaultInitializable {}
+extension Int: _DefaultInitializable {}
+extension Int8: _DefaultInitializable {}
+extension Int16: _DefaultInitializable {}
+extension Int32: _DefaultInitializable {}
+extension Int64: _DefaultInitializable {}
+extension Set: _DefaultInitializable {}
+extension String: _DefaultInitializable {}
+extension Substring: _DefaultInitializable {}
+extension UInt: _DefaultInitializable {}
+extension UInt8: _DefaultInitializable {}
+extension UInt16: _DefaultInitializable {}
+extension UInt32: _DefaultInitializable {}
+extension UInt64: _DefaultInitializable {}
```

Comment:
All of these are also covered by the following: 
`AdditiveArithmetic`
`ExpressibleByArrayLiteral`
`ExpressibleByBooleanLiteral`
`ExpressibleByDictionaryLiteral`
`ExpressibleByUnicodeScalarLiteral`
`RangeReplaceableCollection`